src/get_context_sequences.py

```diff
@@ -0,0 +1,215 @@
+from lxml import etree
+import argparse
+from pyriksdagen.utils import elem_iter, protocol_iterators
+import re
+import progressbar
+import pandas as pd
+
+####    utility functions   ####
+def cleanup_segments(text_seq):
+    # function to remove whitespace from string to get comparable text between corpus and kblab
+    text_seq = text_seq.translate({ord('\n'): ' '})
+    text_seq = text_seq.split(' ')
+    text_seq_list = [s for s in text_seq if s != '']
+    text_seq_string = ' '.join(text_seq_list).strip()
+    return text_seq_string
+
+####    functions to create context sequences from multiple strings ####
+def concat_sequences_left(previous_sequence, current_sequence, target_length = 120):
```

Comment:
Doing this is equivalent to doing concat_sequences_full, right? (maybe with a longer max length). 

We should try and remove one of these functions, and then apply an operation to skip the context after. This removes surface area and makes it easier to maintain and debug.

However, when you make this change, run the original code and check that you get _exactly_ the same result after doing it. Otherwise our training data no longer matches the data we apply this on.